test/com/google/javascript/jscomp/CollapsePropertiesTest.java

```diff
@@ -2093,16 +2089,4 @@ public void testExpressionResultReferenceWontPreventCollapse() {
         + "ns$Outer;\n"
         + "var ns$Outer$Inner=function(){};\n");
   }
-
-  public void testNoCollapseWithInvalidEnums() {
-    test("/** @enum { { a: { b: number}} } */"
-        + "var e = { KEY1: { a: { /** @nocollapse */ b: 123}},\n"
-        + "  KEY2: { a: { b: 456}}\n"
-        + "}", null, null, CollapseProperties.INVALID_NOCOLLAPSE);
-
-    test("/** @enum */ var e = { A: 1, B: 2 };\n"
-        + "/** @type {{ c: { d: number } }} */ e.name1 = {"
-        + "  c: { /** @nocollapse */ d: 123 } };", null, null,
-        CollapseProperties.INVALID_NOCOLLAPSE);
-  }
 }
```

Comment:
Add test with an enum used as a namespace (with properties added to the enum outside the object literal). That's what created the need for this PR.